src/shared/components/ChannelPreview/ChannelPreviewBase.style.tsx

```diff
@@ -34,14 +36,19 @@ export const InnerContainer = styled.div<InnerContainerProps>`
   color: ${colors.gray[300]};
 
   width: calc(156px + calc(2 * ${containerPadding}));
-  padding: 0 ${containerPadding};
+  padding: 0 ${containerPadding} ${sizes(3)} ${containerPadding};
 
   display: flex;
   flex-direction: column;
   align-items: center;
+  border: 1px solid transparent;
   ${hoverTransition}
 `
 
+export const Anchor = styled.a`
+  all: unset;
```

Comment:
What does this do? I've never seen this CSS property used before

Reply:
it resets all of the selected element’s properties; in this case, I use it to strip all the base styles from the `a` tag that we don't need

Reply:
Huh, cool to know :+1: